.github/workflows/cd.yml

```diff
@@ -0,0 +1,28 @@
+name: Conda Publish
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  conda_deployment:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: '3.9'
+          environment-file: .conda/build_env.yaml
+          auto-update-conda: false
+          auto-activate-base: false
+          show-channel-urls: true
+      - name: Build and upload the conda packages
+        uses: uibcdf/action-build-and-upload-conda-packages@v1.2.0
+        with:
+          python-version: '3.9'
+          meta_yaml_dir: .conda
+          user: bschroeter
```

Comment:
wrong user.

Reply:
Referenced in https://github.com/CABLE-LSM/meorg_client/issues/38 to remind myself to do this once this PR is closed.

Reply:
Personally, I would have found it useful to be stated in the PR description. Which I conscientiously read before reviewing code.

Reply:
Noted. This is a good suggestion all around to put more detail in our PR descriptions. Thanks.